client/js/libs/handlebars/tz.js

```diff
@@ -1,17 +1,7 @@
 "use strict";
 
-module.exports = function(time) {
-	time = new Date(time);
-	var h = time.getHours();
-	var m = time.getMinutes();
-
-	if (h < 10) {
-		h = "0" + h;
-	}
+const moment = require("moment");
 
-	if (m < 10) {
-		m = "0" + m;
-	}
-
-	return h + ":" + m;
+module.exports = function(time) {
+	return moment(time).format("HH:mm");
```

Comment:
As long as it's a "Show seconds" checkbox and not a time format text input, then it should be fine 😄